app/assets/stylesheets/_variables.scss

```diff
@@ -18,6 +12,7 @@ $header-line-height: 1.25;
 // Other Sizes
 $base-border-radius: 3px;
 $base-spacing: $base-line-height * 1em;
+$small-spacing: $base-spacing / 2;
```

Comment:
If we’re adding a `$small-spacing` variable, should we also add `$large-spacing` (with a value of `$base-spacing * 2`)?